tests/functional/testplan/runners/pools/func_pool_base_tasks.py

```diff
@@ -1,24 +1,21 @@
 """Base Testplan Tasks shared by different functional tests."""
 
 import os
-from pathlib import Path
-
-import psutil
 import tempfile
+from pathlib import Path
 
-
-from testplan.report import Status
-from testplan.common.utils.path import fix_home_prefix
-from testplan.testing.multitest import MultiTest, testsuite, testcase
-from testplan.testing.multitest.base import MultiTestConfig
 from testplan.common.utils.strings import slugify
+from testplan.testing.multitest import MultiTest, testcase, testsuite
+from testplan.testing.multitest.base import MultiTestConfig
 
 
 @testsuite
 class MySuite:
```

Comment:
Can we de-dup MySuite between this file and test_pool_base? (and get_mtest too)

Reply:
As discussed offline, we will keep both since one represents testsuite in current module and the other for testsuite in imported module.